src/index.ts

```diff
@@ -0,0 +1,102 @@
+export const booleanCalculator = (input:string):boolean | void => {
+    let evaluatedStatement = [];
+    input = formatInput(input);
+    
+    let inputArray: string[] = input.split(" ");
+    evaluatedStatement = evaluator(inputArray);
+    if (input == "NOT TRUE" || input == "FALSE") {
+        return false;
+    } else if (input == "NOT FALSE" || input == "TRUE") {
+        return true;
+    } else if (evaluatedStatement[0] == "TRUE") {
+        return true;
+    } else if (evaluatedStatement[0] == "FALSE") {
+        return false;
+    }
+}
+
+const formatInput = (input:string):string => {
+    let formattedInput:string[] = Array.from(input);
+    for (let i=0; i<formattedInput.length; i++) {
+        if (formattedInput[i] == '(') {
+          formattedInput.splice(i + 1,0, ' ');
```

Comment:
This splice method is pretty difficult to understand. 

And I think this whole function could be much simpler

```
const formatInput = (input) => input.replace("(", "( ").replace(")", " )");
```

Note: using `String.replaceAll` would help solve your multiple brackets issue